docs/mobile-apps/automated-testing/appium/real-devices.md

```diff
@@ -166,6 +166,17 @@ For native app tests on real devices, you must provide a location from which you
 "appium:app","https://github.com/test-apps/ios-app.ipa"
 ```
 
+You can also install an app during a test by using `mobile: installApp` command. 
+
+:::note Limitations
```

Comment:
I would change the order and adjust it a bit to this maybe?


* The provided app path needs to be publicly available, this method does not have access to your local path/local storage
* This method does not have access to apps in our Sauce Storage. Only apps that are publicly available can be installed with this command.
* Because this method doesn't have access to our Sauce Storage we also can't re-sign, see below, and instrument the app. All instrumentation, see [App Settings](https://docs.saucelabs.com/mobile-apps/live-testing/live-mobile-app-testing/#app-settings) options will not work for apps installed using `mobile: installApp` command.
* This method will not work by default for iOS due to signing. Each iOS app needs to be resigned so it is allowed to be installed on our devices. In order to make this work you must use a private device and add UDID of the private device to the provisioning profile for iOS.